apps/fluent-tester/src/theme/CustomThemes.ts

```diff
@@ -32,7 +32,7 @@ export const lightnessOptions = [
 ];
 
 export class TesterThemeReference extends ThemeReference {
-  private _themeName: ThemeNames = 'Default';
+  private _themeName: ThemeNames = Platform.OS === ('win32' as any) ? 'Office' : 'Default';
```

Comment:
Is there a type for platforms (not to use any)?

Reply:
This is expected as win32 isn't a public platform for RN. So this conversion is necessary for the code to accept win32